tests/lookup/tests.py

```diff
@@ -956,3 +957,15 @@ def test_nested_outerref_lhs(self):
             ),
         )
         self.assertEqual(qs.get(has_author_alias_match=True), tag)
+
+    def test_exact_selected_field_rhs_subquery(self):
+        """Ensures GROUP BY in query persists when used as subquery."""
+        _ = Author.objects.create(name='one')
```

Comment:
You can avoid the assignment.
```suggestion
        Author.objects.create(name='one')
```